smithy-aws-protocol-tests/model/awsQuery/xml-lists.smithy

```diff
@@ -31,7 +31,7 @@ operation XmlLists {
 apply XmlLists @httpResponseTests([
     {
         id: "QueryXmlLists",
-        documentation: "Serializes XML lists",
+        documentation: "Deserializes XML lists",
```

Comment:
Note that these tests are not client-specific, so there's no need to change this language. If this is a concern, then I suggest we start using sentences like "Tests for XML list serialization".